frontend/libs/studio-components/src/components/StudioDropdown/StudioDropdownFileUploaderButton/StudioDropdownFileUploaderButton.tsx

```diff
@@ -14,17 +14,22 @@ export type StudioDropdownFileUploaderButtonProps = {
   iconPlacement?: IconPlacement;
   onFileUpload?: (file: File) => void;
   fileInputProps?: FileInputProps;
+  uploadButtonText?: string;
 } & Omit<DropdownButtonProps, 'icon'>;
 
 export function StudioDropdownFileUploaderButton({
-  children,
   icon,
   iconPlacement = 'left',
   onFileUpload,
   fileInputProps,
+  uploadButtonText,
+  disabled,
   ...rest
 }: StudioDropdownFileUploaderButtonProps): ReactElement {
   const { setOpen } = useStudioDropdownContext();
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  const fileInputDisabled: boolean = fileInputProps?.disabled ?? disabled ?? false;
```

Comment:
Since we often use the `is` prefix for booleans elsewhere in the code base, it would be nice to use it here as well.
```suggestion
  const isFileInputDisabled: boolean = fileInputProps?.disabled ?? disabled ?? false;
```